### DISCUSSION TITLE
How to customize blockquotes in pdf?

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#2879] [Q&A]
For the format I am working, I need blockquotes to:
-  begin 4cm from the start of the paragraph;
-  use a smaller font
- use a single space line height (while the main text is 1.5)

How can I customize that?

**Comment by melissavanbussel** [comment]
You can modify the look of the blockquotes using a custom CSS file, by modifying the `blockquote` class!

Example .qmd file:

```
---
title: "Your title goes here"
format: html
css: style.css
---

# This is a heading

This is some text.

:::{.blockquote}

This is a blockquote that contains some words. These are even more words. Blockquotes can contain a lot of words, or not a lot of words.

:::
```

In the style.css file: 

```
.blockquote {
              font-size: 16px;
              line-height: 66%;
              margin-left: 4cm;
}
```

This produces the following:

<img width="397" alt="image" src="https://user-images.githubusercontent.com/28693927/199129563-9914963c-5092-4701-a9f0-ca2552f42ba3.png">

Whereas the default behavior (no custom CSS) looks like this:

<img width="393" alt="image" src="https://user-images.githubusercontent.com/28693927/199129601-7dfee164-733c-4837-9906-9902b6acc3c0.png">

**Reply by mcanouil** [reply]
This won't work for PDF.

**Reply by fredguth** [reply]
I forgot to update this post,I changed the css class for html and redefined `quote`  in [Tufte-Quarto](http://fredguth.github.io/tufte-quarto/)

**Comment by mcanouil** [comment]
You'll have to redefine the `quote` environment and include that in your LaTeX header.
There are a lot of examples online, such as <https://tex.stackexchange.com/questions/325695/how-to-style-blockquote>.
